frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -1,37 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik+Puddles&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pangolin&family=Permanent+Marker');
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bangers&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lobster+Two&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Shantell+Sans&display=swap');
+
 .mainContainer {
   display: flex;
   flex-direction: column;
   align-content: center;
   flex-wrap: wrap;
+  background-image: url('file:///D:/Jeopardy%20-%20Fdipardy/App/Fdipardy/frontend/src/images/space-galaxy-background-free-vector.webp');
+  background-size: cover;
+  background-position: center;
+  height: 754px;
 }
 
 .questionsContainer {
   display: flex;
 }
 
 .category {
+  font-family: 'Frank Ruhl Libre', serif;
+  font-style: oblique;
   display: flex;
   flex-direction: column;
   align-items: center;
+  font-size: 30px;
+  color: #c793c7;
+  margin-right: 32px;
+  margin-top: 0px;
+  text-shadow: 6px 4px 22px  #d1ba68;
+  letter-spacing: normal;
+  animation: flip 5s infinite;
+  animation-delay: calc(.15s * var(--i))
+}
+
+@keyframes flip {
+  0%,80% {
```

Comment:
Learned something new here, I haven'r encountered keyframes before. From what I understand the percentages indicate the start (0% ) and finish(100%) of an animation's transition. Can you explain to me what 0%, 80% here means.  When exactly does the rotation end here?

Reply:
@vesetok you should reply in the relevant conversation. I have asked what exactly does 0%,80%, as in I have asked for a bit of clarification when exactly will the rotation happen. I don't need the MDN docs clarification.

Reply:
@LenkoIvanov These percentages refer to the state of the animation, for example the start state of the animation is written as 0% and the end state as 100%